taskcluster/fpm_taskgraph/transforms/signing.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Copied from https://github.com/mozilla-mobile/fenix/blob/d918cfdf3841a17545053dca2e7480c648ab4f95/taskcluster/fenix_taskgraph/transforms/signing.py. I removed the `worker-type` `dep-signing` in Fenix was made for signing raptor. We don't have this requirement here. I simplified it here.